en/docs/administer/logging-and-monitoring/monitoring/monitoring-server-health.md

```diff
@@ -32,7 +32,7 @@ This section guides you through deploying the Carbon Health Check components in
 
 5. Copy the webapp `api#health-check#v1.0.war`, which is found in the `<HEALTH_CHECK_HOME>/components/org.wso2.carbon.healthcheck.api.endpoint/target/` directory and paste it in the `<API-M_HOME>/repository/deployment/server/webapps` directory.
 
-6. Copy the `health-check.config.xml` configuration file found in the `<API_HOME>/features/org.wso2.carbon.healthcheck.server_1.0.0` directory to your `<PRODUCT_HOME>/repository/conf/` directory.
+6. Copy the `health-check-config.xml` configuration file found in the `<HEALTH_CHECK_HOME>/features/org.wso2.carbon.healthcheck.server.feature/resources` directory to your `<PRODUCT_HOME>/repository/conf/` directory.
```

Comment:
Will changing the `health-check-config.xml` work fine for this? Isn't it the j2 or a toml file that we should be editing?

Reply:
Ack, marking this as resolved.